osu.Game/Screens/OnlinePlay/Components/RoomManager.cs

```diff
@@ -67,11 +57,10 @@ public virtual void CreateRoom(Room room, Action<Room> onSuccess = null, Action<
             {
                 joinedRoom.Value = room;
 
-                update(room, result);
-                addRoom(room);
+                AddOrUpdateRoom(result);
+                room.CopyFrom(result); // Also copy back to the source model, since this is likely to have been stored elsewhere.
 
-                RoomsUpdated?.Invoke();
```

Comment:
Should this call have been removed?

Reply:
It's done as part of `AddOrUpdateRoom()`.